spring-batch-infrastructure/src/main/java/org/springframework/batch/item/file/MultiResourceItemWriter.java

```diff
@@ -141,19 +141,19 @@ public void setSaveState(boolean saveState) {
 		this.saveState = saveState;
 	}
 
-    @Override
+	@Override
 	public void close() throws ItemStreamException {
-                super.close();
+				super.close();
```

Comment:
Why are you replacing 2 tabs with 4 tabs here? The code is not correctly indented after your change.

Reply:
The original code didn't contain 2 tabs but 8 eight spaces and I've replaced each group of 4 spaces with a tab. Anyway, as per your other comment, I'll roll back formatting changes.